src/ResourceManager/AzureBackup/Commands.AzureBackup/Models/AzureBackupContainer.cs

```diff
@@ -23,24 +23,51 @@ namespace Microsoft.Azure.Commands.AzureBackup.Cmdlets
     /// <summary>
     /// Represents Azure Backup Container
     /// </summary>
-    public class AzureBackupContainer : AzureBackupVaultCmdletBase
+    public class AzureBackupContainer : AzureBackupVaultContextObject
     {
+        /// <summary>
+        /// Type of the Azure Backup container
+        /// </summary>
         public string ContainerType { get; set; }
```

Comment:
We should think a way of letting user know the applicable values here.
Same for Status, HealthStatus.